cmd/zed/dev/vcache/agg/command.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/brimdata/zed/pkg/charm"
 	"github.com/brimdata/zed/pkg/storage"
 	"github.com/brimdata/zed/runtime/vam"
+	vamop "github.com/brimdata/zed/runtime/vam/op"
```

Comment:
Nit: No need to rename this import since `op` doesn't conflict with any of the others.
```suggestion
	"github.com/brimdata/zed/runtime/vam/op"
```